README.md

```diff
@@ -1,6 +1,7 @@
-Linux: [![Build Status](https://travis-ci.org/GENIVI/CANdevStudio.svg?branch=master)](https://travis-ci.org/GENIVI/CANdevStudio) <br />
-Windows: [![Build status](https://ci.appveyor.com/api/projects/status/y7vacbyvso12ud6e/branch/master?svg=true)](https://ci.appveyor.com/project/rkollataj/candevstudio-qflna/branch/master) <br />
-Coverage: [![codecov](https://codecov.io/gh/GENIVI/CANdevStudio/branch/master/graph/badge.svg)](https://codecov.io/gh/GENIVI/CANdevStudio)
+Linux: [![Build Status](https://travis-ci.org/PrzemyslawDraszkiewicz/CANdevStudio.svg?branch=master)](https://travis-ci.org/PrzemyslawDraszkiewicz/CANdevStudio.svg?branch=master) <br />
```

Comment:
Please remove this change from PR. We will keep original badges fro the whole project duration